### ISSUE TITLE
Protractor-Recorder Stopped Working

### ISSUE THREAD

**Issue by shanethughes** [ealves/protractor-recorder#39]
Hey ealves,

Maybe you can help me out here. All of a sudden after successfully being able to use Protractor-Recorder the last couple of days - it stopped working today. I have uninstall and installed ~5 times. I can get the app running and the Selenium instance to fire but no recording takes place. Has this happened to you before?

**Comment by ealves** [comment]
Hi,

I don't remember if it happened to me before, but try clear your local storage too.

**Comment by shanethughes** [comment]
By local storage are you referring to webdriver-manager? If so I tried to run webdriver-manager clean and I receive an error 'operation not permitted - unlink *path to global Selenium*/chromedriver_2.22.exe'.

**Comment by ealves** [comment]
No, i was referring to the browser's local storage. If you are using Google Chrome, go to DevTools, Application tab, Storage and Local Storage. Try deleting all the keys.

**Comment by CliveBiddulph** [comment]
Gents, I would like to thank you for your wisdom, but I had an major brain fail, and the error lies with the user :-(   not the great product :-) .
